src/Workspaces/CSharp/Portable/CodeStyle/CSharpCodeStyleOptions.cs

```diff
@@ -154,6 +154,12 @@ internal static class CSharpCodeStyleOptions
                 EditorConfigStorageLocation.ForBoolCodeStyleOption("csharp_prefer_inferred_anonymous_type_member_names"),
                 new RoamingProfileStorageLocation($"TextEditor.CSharp.Specific.{nameof(PreferInferredAnonymousTypeMemberNames)}")});
 
+        public static readonly Option<CodeStyleOption<bool>> PreferNamingLiteralArguments = new Option<CodeStyleOption<bool>>(
+            nameof(CodeStyleOptions), nameof(PreferNamingLiteralArguments), defaultValue: CodeStyleOptions.TrueWithSuggestionEnforcement,
+            storageLocations: new OptionStorageLocation[] {
+                EditorConfigStorageLocation.ForBoolCodeStyleOption("csharp_prefer_naming_literal_arguments"),
```

Comment:
dotnet_style_prefer_named_literal_arguments (since this will be supported for Vb right?  i thought you said VB had non-trailing arg support).

That said, i think i would prefer that this was:

dotnet_style_prefer_named_arguments=always,never,for_literals,for_... (i.e. for_lambdas, for_etc).  

This leaves us open for extending this in the future.  bools make that harder.

Reply:
If supported for VB, should move to CodeStyleOptions